src/main/java/org/kohsuke/github/GHDiscussion.java

```diff
@@ -0,0 +1,58 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * A discussion in GitHub Team.
+ *
+ * @author Charles Moulliard
+ */
+public class GHDiscussion extends GHObject {
+
+    protected GitHub root;
+    protected GHOrganization organization;
+    protected GHTeam team;
+    protected String body, title, html_url;
+
+    @Override
+    public URL getHtmlUrl() throws IOException {
+        return GitHubClient.parseURL(html_url);
+    }
+
+    public GHDiscussion wrapUp(GHTeam team) throws IOException {
+        this.root = team.root;
+        this.organization = team.getOrganization();
+        this.team = team;
+        return this;
+    }
+
+    public GHDiscussion wrapUp(GitHub root) {
+        this.root = root;
+        return this;
+    }
+
+    public GHDiscussion wrapUp(GHOrganization owner) {
+        this.organization = owner;
+        this.root = owner.root;
+        return this;
```

Comment:
```suggestion
    public GHDiscussion wrapUp(GHOrganization owner) {
        this.organization = owner;
        return this;
```

Reply:
Can a discussion exist for an organization but not a team?  No right? 

You'll have `root` already.   Let's drop `owner` and keep `team`, and add a `getTeam()` method that handles the wrapping in a `refresh(team)` call.

Reply:
This is correct. Only discussions belong to teams and not org.